x/group/client/testutil/tx.go

```diff
@@ -616,6 +584,8 @@ func (s *IntegrationTestSuite) TestTxUpdateGroupMembers() {
 	}]}`, val.Address.String(), tooLongMetadata)
 	invalidMembersMetadataFileName := testutil.WriteToNewTempFile(s.T(), invalidMembersMetadata).Name()
 
+	s.fundAllGroupPolicies("2", 1000)
```

Comment:
Some proposals for this helper:
- rename it to `fundGroupPolicy`? It only funds one policy afaict
- why is the groupID a string?
- i think it would be clearer if the 2nd arg was a `sdk.Coins`

Reply:
Actually it iterates over al the group policies of a groupID

Reply:
Ah yeah sounds good 👍